agent-install/agent-install.sh

```diff
@@ -43,13 +43,16 @@ where:
     -c          - path to a certificate file
     -k          - path to a configuration file (if not specified, uses agent-install.cfg in current directory, if present)
     -p          - pattern name to register with (if not specified, registers node w/o pattern)
-    -i          - installation packages location (if not specified, uses current directory)
+    -i          - installation packages location (if not specified, uses current directory). if the argument begins with 'http' or 'https', will use as an apt repository
+    -j          - file location for the public key for an apt repository specified with '-i'
     -n          - path to a node policy file
     -s          - skip registration
     -v          - show version
     -l          - logging verbosity level (0-5, 5 is verbose)
     -u          - exchange user authorization credentials
     -f          - install older version without prompt. overwrite configured node without prompt.
+    -w 		- specify a service to wait to return control until it starts on the node
```

Comment:
Please change the wording of -w to "wait for the named service to start executing on this node"